### PR TITLE
charts/victoria-metrics-anomaly: update vmanomaly, bugfixes

### PR DESCRIPTION
**Pull request by zekker6**
- update vmanomaly to 1.16.1
- update configuration for testing to be able to run vmanomaly
- fix configuration templating
- add option to dump models and data

### CODE REVIEW

**Review by AndrewChubatiuk — APPROVED** [review]

File: charts/victoria-metrics-anomaly/templates/configmap.yaml [hunk 1]
```diff
@@ -1,3 +1,10 @@
+{{- if not .Values.config.reader.tenant_id -}}
+{{- $ := unset .Values.config.reader "tenant_id" }}
+{{- end -}}
+{{- if not .Values.config.writer.tenant_id -}}
+{{- $ := unset .Values.config.writer "tenant_id" }}
+{{- end -}}
+---
```
Comment (fred-navruzov) [inline charts/victoria-metrics-anomaly/templates/configmap.yaml:7]: can you please also add the same for `monitoring.push` as we also have `tenant_id` there ?

Comment (zekker6) [reply charts/victoria-metrics-anomaly/templates/configmap.yaml:7]: It is not included as a part of default values so unless user sets it manually that won't be a problem for vmanomaly.

Comment (fred-navruzov) [reply charts/victoria-metrics-anomaly/templates/configmap.yaml:7]: oh, got it, thanks for clarification!

**Review by fred-navruzov — COMMENTED** [review]
thanks, LGTM!

**Pull request merged**